src/core/common/message.hpp

```diff
@@ -529,6 +530,23 @@ class Message : public Buffer
      */
     bool IsSubTypeMle(void) const;
 
+    /**
+     * This method checks whether this multicast message may be looped back.
+     *
+     * @retval TRUE   If message may be looped back.
+     * @retval FALSE  If message must not be looped back.
+     *
+     */
+    bool GetMulticastLoop(void) const { return GetMetadata().mMulticastLoop; }
```

Comment:
How about `GetAllowMulticastLoop()` and `SetAllowMulticastLoop()`?

Reply:
The flag name is from the most common used flag IPV6_MULTICAST_LOOP. Is it better to keep it as is?

Reply:
How about `IsMulticastLoopEnabled()` since it's `bool`?

Reply:
I am fine with it as is.
Also like the @Moandor-y suggestion.

Reply:
@bukepo , thoughts on `IsMulticastLoopEnabled()`?

Reply:
I'm fine with changing the name, but prefer doing it later so that the PR can be merged without re-running all the CI tests.

Reply:
Another proposal `IsMulticastLoop()` 
- `Loop` itself is descriptive
- seems more match with below `SetMulticastLoop()`

Reply:
Just take another look, if rename the getter to `IsMulticastLoopEnabled()`, what's the proper name for the coresponding setter? If the current flag name is already common used, I prefer to keep it as is. Thoughts? @abtink @Moandor-y @jwhui @librasungirl

Reply:
I am still fine with it as is :)

If you want to change it, for setter name, we can follow existing pattern from `IsLinkSecurityEnabled()` / `SetLinkSecurityEnabled()` or `IsTimeSync()` / `SetTimeSync()`, etc.  So we can go for `IsMulticastLoopEnabled()` and `SetMulticastLoopEnabled()`.

Reply:
No strong preference, fine to me to keep it as is.

Reply:
I prefer `IsMulticastLoopEnabled()` and `SetMulticastLoopEnabled()`. I'm fine with it as is too.